elasticsearch-model/spec/elasticsearch/model/naming_inheritance_spec.rb

```diff
@@ -19,105 +19,185 @@
 
 describe 'naming inheritance' do
 
-  before(:all) do
-    class ::TestBase
-      extend ActiveModel::Naming
+  context 'without using proxy' do
```

Comment:
I have just move d the tests into a context and changed the way the classes are defined so that they can be redefined with a different superclass in the other context.